index.js

```diff
@@ -13,6 +13,10 @@ function isFQDN(_str, {
 	allowUnderscores = false,
 	allowTrailingDot = false
 } = {}) {
+	if (typeof _str !== 'string') {
```

Comment:
Or, what about, converting provide args into str before validation?

Reply:
Would probably be slower than this simple type check.